pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+	"github.com/spf13/cobra"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	newcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+)
+
+const (
+	newBuildLong = `Create a new build in OpenShift by specifying source code, and/or images.
+
+This command will try to create a BuildConfig for your application using images, or code that has
+a public repository located on your system. It will lookup the images on the
+local Docker installation (if available), a Docker registry, or an OpenShift image stream.
+If you specify a source code URL, it will set up a build that takes your source code and converts
+it into an image that can run inside of a pod. Local source must be in a git repository that has
+remote repository that the OpenShift instance can see.
+
+Once the BuildConfig will be created you may need to run a build with 'start-build'.`
```

Comment:
Throughout all the logs/msgs we keep original object name, thus here I'll leave BuildConfig.